.github/dependabot.yml

```diff
@@ -136,3 +136,10 @@ updates:
     allow:
       - dependency-name: "bitnami/kafka"
         update-types: ["version-update:semver-minor"]
+  - package-ecosystem: docker
+    directory: /docker-compose/cassandra
```

Comment:
@hellspawn679 I don't think this is working, the dependabot runs did not pick this up. I suspect that when you pick `docker` ecosystem and a directory, dependabot looks for standard file names like `Dockerfile` or `docker-compose.yml`. Since you're using custom file names like `v3.yml`, dependabot doesn't know that they are for docker-compose. We probably need to list the files explicitly instead of specifying a dir only.

Reply:
https://github.com/jaegertracing/jaeger/issues/5552